client/src/pages/CredentialsPage/CredentialsPage.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
For this particular page, I would suggest that this component should be in components (since it doesn't make an actual page) rather just a larger component (which contains components).

In this case, I would copy over line 26 onwards as a component.
Then in the Login/Signup page, I would have it as

```
<div className={styles.bg}>
    <CredentialsForm/>
</div>
```

Although the outer div will be repeated for both the Login And Signup page, it provides more reusability for the CredentialsForm as a component itself (since it won't be tied down to a particular background as backgrounds are mainly for the page itself rather than a component)

Reply:
Fixed: moved background div to actual page and moved `CredentialsForm` to `components/`